src/server/services/govuk-notify.ts

```diff
@@ -141,7 +153,183 @@ export async function sendEditDetailsEmail(
+      emailAddress = "ali@cautionyourblast.com";
+    }
+
+    const notifyTemplates: Record<number, string> = {
+      28: GOVUK_NOTIFY_ANNUAL_REVIEW_POST_ONE_MONTH_NOTICE ?? "",
+      7: GOVUK_NOTIFY_ANNUAL_REVIEW_POST_ONE_WEEK_NOTICE ?? "",
+      1: GOVUK_NOTIFY_ANNUAL_REVIEW_POST_ONE_DAY_NOTICE ?? "",
+      0: GOVUK_NOTIFY_ANNUAL_REVIEW_POST_STARTED ?? "",
+    };
+
+    const notifyTemplate = notifyTemplates[daysBeforeAnnualReviewStart];
+    logger.info(
+      `personalisation - template ${notifyTemplate}, typePlural: ${typePlural}, country: ${country}, annualReviewDate: ${annualReviewDate}, daysBeforeAnnualReviewStart: ${daysBeforeAnnualReviewStart}`
+    );
+    await getNotifyClient().sendEmail(
+      notifyTemplate,
+      emailAddress,
+      {
+      personalisation: {
+        typePlural,
+        country,
+        annualReviewDate,
+        typePluralCapitalised: typePlural.toUpperCase(),
+      },
+    });
+  } catch (error) {
+    return { error: new Error(`Unable to send change request email: ${error.message}`) };
+  }
+  return { result: true };
+}
+
+export async function sendAnnualReviewProviderEmail(
+  daysBeforeAnnualReviewStart: number,
+  emailAddress: string,
+  typePlural: string,
+  country: string,
+  contactName: string,
+  deletionDate: string,
+  changeLink: string
+): Promise<{ result?: boolean, error?: Error }> {
+  try {
+    if (config.isSmokeTest) {
+      logger.info(`isSmokeTest[${config.isSmokeTest}], would be emailing to ${emailAddress}`);
+      return { result: true };
+    }
+
+    if (emailAddress !== "ali@cautionyourblast.com") {
+      emailAddress = "ali@cautionyourblast.com";
+    }
+
+    logger.info(
+      `personalisation -  contactName:${contactName}, typePlural: ${typePlural}, country: ${country}, deletionDate: ${deletionDate}, changeLink: ${changeLink}`
+    );
+
+    await getNotifyClient().sendEmail(
+      GOVUK_NOTIFY_ANNUAL_REVIEW_PROVIDER_STARTED,
+      emailAddress,
+      {
+      personalisation: {
+        contactName,
+        typePlural,
+        country,
+        deletionDate,
+        changeLink,
+      },
+    });
+  } catch (error) {
+    return { error: new Error(`Unable to send annual review provider email: ${error.message}`) };
+  }
+  return { result: true };
+}
+
+export async function sendUnpublishedPostEmail(
+  daysBeforeUnpublished: number,
+  emailAddress: string,
+  typePlural: string,
+  country: string,
+  numberNotResponded: string
+): Promise<{ result?: boolean, error?: Error }> {
+  try {
+    if (config.isSmokeTest) {
+      logger.info(`isSmokeTest[${config.isSmokeTest}], would be emailing to ${emailAddress}`);
+      return { result: true };
+    }
+    logger.info(`sending sendUnpublishedPostEmail for ${daysBeforeUnpublished} days before being unpublished
+    [typePlural: ${typePlural}, country: ${country}, number not responded: ${numberNotResponded}]`)
+
+    const notifyTemplates: Record<number, string> = {
```

Comment:
I think either the variable name, or the record should be descriptive. 
`daysLeftToUnpublishToTempalteMap` or `Record<Warning, string>`